roles/openshift_deploy_registry/defaults/main.yml

```diff
@@ -0,0 +1,2 @@
+---
+# defaults file for openshift_deploy_registry
```

Comment:
I am prob going to squash the commits, this is just so I can run it through our jenkins testing.